java/client/src/main/java/glide/api/models/commands/geospatial/GeoSearchOptions.java

```diff
@@ -0,0 +1,90 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.geospatial;
+
+import glide.api.commands.GeospatialIndicesBaseCommands;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+
+/**
+ * Optional arguments for {@link GeospatialIndicesBaseCommands#geosearch(String,
+ * GeoSearchOrigin.SearchOrigin, GeoSearchShape, GeoSearchOptions)} command, options include:
+ *
+ * <ul>
+ *   <li>WITHDIST: Also return the distance of the returned items from the specified center point.
+ *       The distance is returned in the same unit as specified for the <code>searchBy
+ *           </code> argument.
+ *   <li>WITHCOORD: Also return the coordinate of the returned items.
+ *   <li>WITHHASH: Also return the geohash of the returned items. for the general user.
+ * </ul>
+ *
+ * @see <a href="https://redis.io/commands/geosearch/">redis.io</a>
+ */
+@Builder
+public final class GeoSearchOptions {
+    /** Redis API keyword used to perform geosearch with coordinates. */
```

Comment:
Remove Redis reference.

Reply:
responded in https://github.com/aws/glide-for-redis/pull/1685#discussion_r1659426140